scripts/install_texlive.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+if [[ -x "/usr/bin/latex" ]]; then
+    echo "texlive already installed"
+    exit 0
```

Comment:
Why skip all other processes? In that case, the package installation will not take place, right?

Reply:
I don't believe any of the processes listed inside `install_texlive.sh` ought to take place if we're using the ubuntu binaries which already supply these.